forage-android/src/main/java/com/joinforage/forage/android/core/element/state/ElementState.kt

```diff
@@ -12,6 +22,10 @@ interface ElementState {
     val validationError: ElementValidationError?
 }
 
+/**
+ * An interface that represents the state of a
```

Comment:
@djoksimo I'm open to this! Does this look like what you had in mind? 

```
{
  "isFocused": true,
  "isBlurred": false,
  "isEmpty": false,
  "isValid": true,
  "isComplete": true,
  "validationError": null
}
```

If not I'd appreciate it if you could share your ideal example 🙏  I asked ChatGPT in case I was missing something.

Reply:
Sry, I just meant to mention a brief summary of what `PinElement` state includes: 

> An interface that represents the state of a .... . This includes input validation values....

But given that you link to `ElementState` interface, I don't think this is necessary! So no action needed here